lms/static/js/learner_dashboard/program_list_factory.js

```diff
@@ -0,0 +1,21 @@
+;(function (define) {
```

Comment:
The page is called Program Listing page.
I was naming this according to the page itself. 
Do you have a better suggestion?